public/steelthreads/json/en/thread-0.json

```diff
@@ -3,64 +3,147 @@
+  "estimatedTime": 5,
+  "stepDoc": "creating-addresses.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc",
+  "attributes": {
+  "launcher-url": "http://launcher-launcher.apps.city.openshiftworkshop.com/"
+  }
+  },
+  {
+  "title": "Running a Node.js messaging app",
+  "description": "The Red Hat Developer Launcher enables you to build and deploy applications on OpenShift. In this procedure, you deploy an example Node.js application that is pre-configured to provide a REST API that returns a list of fruit.",
+  "estimatedTime": 5,
+  "stepDoc": "running-node-app.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc",
+  "attributes": {
+  "launcher-url": "http://launcher-launcher.apps.city.openshiftworkshop.com/"
+  }
+  },
+  {
+  "title": "Editing the Node.js messaging app",
+  "description": "Che allows you edit the source code for your Node.js app in the browser.",
+  "estimatedTime": 5,
+  "stepDoc": "editing-node-app-with-che.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc",
+  "attributes": {
+  "launcher-url": "http://launcher-launcher.apps.city.openshiftworkshop.com/"
+  }
+  },
+  {
+  "title": "Running a Spring Boot CRUD app",
+  "description": "In this procedure, you deploy an example Spring Boot CRUD application that is pre-configured to act as a simple grocery inventory application.",
+  "estimatedTime": 5,
+  "stepDoc": "running-spring-app.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc",
+  "attributes": {
+  "launcher-url": "http://launcher-launcher.apps.city.openshiftworkshop.com/"
+  }
+  },
+  {
+  "title": "Creating AMQP connection in Fuse Ignite",
+  "description": "To receive messages from EnMasse, you must create a connection in Fuse Ignite.",
+  "estimatedTime": 5,
+  "stepDoc": "creating-amqp-connection-in-fuse.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc"
+  },
+  {
+  "title": "Creating HTTP connection in Fuse Ignite",
+  "description": "To send messages to the Spring Boot app, you must create a connection in Fuse Ignite.",
+  "estimatedTime": 6,
+  "stepDoc": "creating-http-connection-in-fuse.adoc",
+  "stepDocInfo": "complete-before-proceeding.adoc"
+  },
+  {
+  "title": "Creating Integration in Fuse Ignite",
+  "description": "To use the Fuse connections, you must create an integration in Fuse Ignite.",
+  "estimatedTime": 5,
+  "stepDoc": "creating-fuse-integration.adoc",
+  "stepDocSuccess": "thread-0-success.adoc"
+  },
+  {
+  "title": "Using the application integration",
+  "description": "After setting up the integration between the Node.js and Spring Boot applications, we can use the integration to create a new fruit for the grocery inventory application.",
+  "estimatedTime": 5,
+  "stepDoc": "using-integration.adoc",
+  "stepDocSuccess": "thread-0-success.adoc"
+  }
   ]
-}
+  }
+  
+  // 20180903163703
+  // https://raw.githubusercontent.com/integr8ly/tutorial-web-app/8921fd954396fe26445553ada441b186d8fa7ca3/public/steelthreads/json/en/thread-0.json
```

Comment:
so these comments are breaking my json validations... can you possible shore up this JSON and make sure it validates? You can use https://jsonlint.com/ for example.

Reply:
not sure if you had some questions or just revising?

Reply:
updated to make valid, but really don't know about the fields in this file, so maybe I still don't have it right. 
eg I'd like to have mulitple  procedures in a task, could stepDoc be an array?

Reply:
sure - we can make stepDoc an array if that is easier (and just render the array of docs). 

Regarding the verification, i don't think it is simple in Ascii format to add the icon spans (having been looking for a solution, but nothing seems simple). For now, I will just add them dynamically on the React side if OK. Will send you a commit shortly...

Reply:
ok pushed a small change here for the alerts to try to meet the design styling... what do you think of this approach? Let me know if OK. https://github.com/priley86/integr8ly-prototype/commit/b090abfe387b7db8a5bd0a2039bc51d05ab674eb